Userland/Libraries/LibGL/Shaders/Shader.cpp

```diff
@@ -20,4 +20,26 @@ ErrorOr<void> Shader::compile()
     return {};
 }
 
+size_t Shader::info_log_length() const
+{
+    if (!m_info_log.has_value())
+        return 0;
+
+    // Per the spec we return the size including the null terminator
```

Comment:
Discuss: to make this absolutely clear, we could name the function `info_log_length_including_null_terminator` so you don't need to read the implementation to understand what's going on. We've done similar things for other Serenity APIs.